platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+meaning or intent, and formatting changes. Examples of material changes include changes to behaviors or APIs, addition
+or removal of content, etc. If any material changes are needed, the PR must be closed, and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR.
+
+---
+
+## Voting on A Proposal
+
+Votes follow the [Apache Voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions)
+scheme. Votes are cast in comments on the PR on a range between -1 to -0, +0 to +1 with the following semantics:
+
+* -1: is a veto.
+* -0 >= x > -1: is a vote generally against the proposal.
+* +0 <= x < +1: is a vote generally in favor of the proposal.
+* +1: is a vote in favor of the proposal.
+
+The magnitude of the numeric value of the vote indicates the strength of the sentiment behind the vote.
+
++1 votes can also be expressed by approving the PR. -1 votes can be expressed by "Requesting Changes" on the PR and
+must be accompanied by a comment explaining the reason for the veto.
+
+---
+
+## Acceptance of A Proposal
+
+A proposal becomes `Accepted` when the following criteria have been fulfilled:
+
+1. The proposal has been in the Voting state for at least 3 business days. Business days are defined as SwirldsLabs
+   working days and excludes weekends and company holidays.
+2. The proposal must have at least three +1 votes from platform code owners.
+3. The proposal must not have any -1 votes from platform code owners or architects.
+
+Once accepted, the proposal must be moved to the Accepted status in
+the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board and implementation of the
+proposal may begin.
+
+---
+
+## Superseding A Proposal
+
+A proposal is superseded by a new proposal when material changes are needed once voting has started. The old proposal
+must link to the new proposal and indicate that it is being superseded. After the proposal has been updated, the PR's
+status in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project should be
+changed to `Superseded`.
+
+---
+
+## Withdrawing A Proposal
+
+A proposal may be withdrawn after voting occurs. The proposal must be updated with a reason for the withdrawal. After
+the proposal has been updated, the PR's status in
+the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project should be changed
+to `Withdrawn`.
+
+---
+
+## Delivery of A Proposal
+
+Once an accepted proposal has been completely implemented, tested, the code merged into `develop`, and the feature is
+enabled for production, the proposal's content should be merged with the documentation of the platform
+in `platform-sdk/docs` and removed from `platform-sdk/docs/proposals`. Once the platform documentation is updated, the
```

Comment:
**oh wait.**

- So a proposal as a text document does not exist in our main branch until it is fully delivered?
- Is there a `delivered-proposals` folder under `platform-sdk/docs`?
- How do I see that a proposal is a proposal and not a documentation for an api?

Reply:
Proposals that have been accepted are documented in `platform-sdk/docs/proposals`
Proposals that are delivered are just part of the platform documentation in `platform-sdk/docs`

Reply:
My thought was that the content would be merged in a way that makes sense with the current documentation. If a core behavior is changes, then existing documentation for it must be updated. If a new feature is added, then perhaps a new folder should be added in the appropriate location.

I do not want to dictate the organization of the `docs` folder here. But at the very least, once the documentation moves out of the `docs/proposals` directory into just `docs`, then it is no longer a proposal and should not be named as such.